build.gradle.kts

```diff
@@ -24,13 +25,25 @@ group = "org.jmailen.gradle"
 description = projectDescription
 
 object Versions {
-    const val androidTools = "4.2.2"
-    const val jetbrainsAnnotations = "22.0.0"
+    const val androidTools = "7.0.4"
     const val junit = "4.13.2"
-    const val ktlint = "0.43.0"
+    const val ktlint = "0.43.2"
     const val mockitoKotlin = "4.0.0"
 }
 
+configurations {
+    register("testRuntimeDependencies") {
+        extendsFrom(compileOnly.get())
+    }
```

Comment:
`compileOnly` isn't resolvable anymore. I explicitly declare new configuration which can be used via `pluginUnderTestMetadata` task